src/js/App/Sidenav/NavigationItem.js

```diff
@@ -3,6 +3,6 @@ import { NavItem } from '@patternfly/react-core/dist/esm/components/Nav';
 
 const basepath = document.baseURI;
 
-export default ({ itemId, title, parent = '', ...props }) => (
-    <NavItem {...props} itemId={itemId} preventDefault to={`${basepath}${parent}/${itemId}`}>{title}</NavItem>
+export default ({ itemID, title, parent = '', ...props }) => (
+    <NavItem {...props} itemID={itemID} preventDefault to={`${basepath}${parent}/${itemID}`}>{title}</NavItem>
```

Comment:
Here it should be `itemId={itemID}` because patternfly navigation expects `itemId`